fastai/callback/azureml.py

```diff
@@ -0,0 +1,46 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/74_callback.azureml.ipynb (unless otherwise specified).
+
+__all__ = ['AzureMLCallback']
+
+# Cell
+import tempfile
+from ..basics import *
+from ..learner import Callback
+
+# Cell
+from azureml.core.run import Run
+
+# Cell
+class AzureMLCallback(Callback):
+    "Log losses, metrics, model architecture summary to AzureML"
+
```

Comment:
Add this line here to fix your smooth loss problem:

```python
order = Recorder.order+1
```

Reply:
Do you mean `self.order`? Just setting order did not work.

Reply:
Ah, I got that wrong. It is set just inside the class and not in a `__init__`. 

Will add the commit in the next minutes.